.python-version

```diff
@@ -0,0 +1 @@
+{{min_python}}
```

Comment:
This must have gotten here by accident when I tried the copier update. Can you remove it?

Reply:
The whole file or the line?